docs/daily-usage.md

```diff
@@ -222,8 +222,8 @@ docker-compose -f prod.yaml up -d
 #### Backups
 
 Backups are only available in the production environment. They are provided by
-[tecnativa/duplicity:postgres-s3](https://github.com/Tecnativa/docker-duplicity). The
-structure of the backed up folder:
+[tecnativa/docker-duplicity-postgres-s3](https://github.com/Tecnativa/docker-duplicity).
```

Comment:
I think you should link to the package, or rename the link label to just `docker-duplicity`.